### PR TITLE
GalliumNine: fix itering shared lib folders (Fixes #2332)

### LINKED ISSUE #2332: Nine: Lutris can't find Nine libraries
A lot of changed before creating PR #2092 and merging it - biggest issue is commit 792c22176eff9e063b22d7b9700e2e9b79a11fae, which changes return val of iter_lib_folders() from lists to strings. I believe I used iter_lib_folders() in #2092 because I needed distinguish between lib32 and lib64 paths. I will take a look at this and try to fix it ASAP (this week).

Original report:
https://github.com/lutris/lutris/pull/2092#issuecomment-529362315

### PR DESCRIPTION
**Pull request by sonicpp**
iter_lib_folders() return strings instead of lists, so we have to
find other way how to distinguish between 32 and 64 bit libraries

### CODE REVIEW

**Comment by soredake** [comment]
Tested, option is still grayed out.

**Comment by sonicpp** [comment]
@soredake hi, the code is currently checking installed gallium nine standalone (looking for files d3d9-nine.dll and ninewinecfg.exe in system folder) and gallium nine support in mesa drivers (looking for file d3dadapter9.so.1). Unavailable option means it can not find some of these files. Can you please post locations of files I mentiond on your system (what is your system btw? Mine is Arch).

Location of these files on my system:
64 bit:
/usr/lib/d3d/d3dadapter9.so.1
/usr/lib/wine/fakedlls/ninewinecfg.exe
/usr/lib/wine/fakedlls/d3d9-nine.dll

32 bit:
/usr/lib32/d3d/d3dadapter9.so.1
/usr/lib32/wine/fakedlls/ninewinecfg.exe
/usr/lib32/wine/fakedlls/d3d9-nine.dll

Thank you very much!

**Comment by soredake** [comment]
/usr/lib/d3d/d3dadapter9.so.1 and /usr/lib32/d3d/d3dadapter9.so.1 is there 
exe and dll is in different location because i use -git version:
```
❯ yay -Ql gallium-nine-git
gallium-nine-git /usr/
gallium-nine-git /usr/bin/
gallium-nine-git /usr/bin/nine-install
gallium-nine-git /usr/share/
gallium-nine-git /usr/share/nine/
gallium-nine-git /usr/share/nine/bin64/
gallium-nine-git /usr/share/nine/bin64/ninewinecfg.exe.fake
gallium-nine-git /usr/share/nine/bin64/ninewinecfg.exe.so
gallium-nine-git /usr/share/nine/lib64/
gallium-nine-git /usr/share/nine/lib64/d3d9-nine.dll.fake
gallium-nine-git /usr/share/nine/lib64/d3d9-nine.dll.so
```

**Comment by sonicpp** [comment]
Sigh...why is every maintainer using different installation destinations?

@soredake Seems like another issue (not sure if we should create new bug for it or keep this one...). Anyway, could you please install *gallium-nine* package instead of *gallium-nine-git* and give it a try (so we could know if there is not another issue)?

In the meantime I will try to make a research on wine-nine packages of several other distributions and locate all installation locations...or I can try to make gallium nine standalone as part of lutris (as dxvk/d9vk already are).

Thanks a lot for your patience!

TLDR - this PR fixes one issue, but there is another one (not fixed yet)

**Comment by soredake** [comment]
Switched to wine-nine, works now with pr, i hope issue with -git version will be resolved.
